geode-core/src/distributedTest/java/org/apache/geode/distributed/LocatorDUnitTest.java

```diff
@@ -1314,17 +1314,21 @@ public void run() {
     vm1.invoke("waitUntilLocatorBecomesCoordinator", this::waitUntilLocatorBecomesCoordinator);
     vm2.invoke("waitUntilLocatorBecomesCoordinator", this::waitUntilLocatorBecomesCoordinator);
 
-    if (vm1.invoke(() -> system.getDistributedMember().equals(getView().getCreator()))) {
-      vm2.invoke(() -> {
-        GeodeAwaitility.await()
-            .until(() -> !system.getDistributedMember().equals(getView().getCreator()));
-      });
-    } else {
-      vm2.invoke(() -> {
-        GeodeAwaitility.await()
-            .until(() -> system.getDistributedMember().equals(getView().getCreator()));
-      });
-    }
+    await().untilAsserted(() -> {
+      MemberIdentifier viewCreator = vm1.invoke(() -> getView().getCreator());
+      MemberIdentifier viewCreator2 = vm1.invoke(() -> getView().getCreator());
+
+      if (isSystemConnected()) {
```

Comment:
If isSystemConnected returns false, that means the test will pass immediately. Is that your intention?

Reply:
uh, nope... I was just guarding against the NPE that was happening...